packages/app-project/stores/initStore.js

```diff
@@ -0,0 +1,31 @@
+import { applySnapshot } from 'mobx-state-tree'
+import {
+  panoptes as panoptesClient,
+  projects as projectsClient
+} from '@zooniverse/panoptes-js'
+import Store from './Store'
+
+let store = null
+
+const client = {
+  panoptes: panoptesClient,
+  projects: projectsClient.projects
```

Comment:
This should be just `projectsClient`

Reply:
I thought that, but the `projects` export from `panoptesjs` includes things like the mocks etc.

Reply:
Hm, yes, but when I was debugging `projectsClient.projects` is undefined. I have it defined as `projects` and `projectsMocks` separately in the lib export: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-panoptes-js/src/index.js

I'm open to changing it to however it makes the most sense.